package.json

```diff
@@ -25,7 +25,7 @@
     "_____": "",
     "fmt": "./ci/dev/fmt.sh",
     "lint": "./ci/dev/lint.sh",
-    "test": "./ci/dev/test.sh",
+    "test": "echo 'Run yarn test:unit or yarn test:e2e' && exit 1",
```

Comment:
Will this affect anything in CI?

Reply:
It shouldn't, unless I forgot to change something in https://github.com/cdr/code-server/pull/2852

Reply:
The file it was running doesn't exist so if it did I think we'd have test failures. :smile: